### DISCUSSION TITLE
Aprimorando conhecimentos como Front-End

### DISCUSSION THREAD

**Discussion by vitocaio** [frontendbr/forum#185] [Discussão]
Ola amigos front ends,
Sou desenvolvedor javascript a pouco tempo, me considero um júnior no assunto e gostaria de me aprimorar ainda mais, na verdade quero me tornar full-stack em javascript, back-end(nodeJs) e front end(angulaJs, etc...). Quero começar entendendo a parte de front, mais a fundo, gostaria de alguns matérias referente ao front end, mas gostaria que fossem mais avacados, encontro muito material para iniciantes, mas poucos para aperfeiçoamento no assunto e boas praticas, ficarei grato se me ajudassem a encontrar materiais para aperfeiçoar meus conhecimentos e me tornar um desenvolvedor mais avançado.

Obrigado.

**Comment by felipefialho** [comment]
Por coincidência, nos últimos dias coloquei dois artigos no meu site:
- [Do Zero a Herói Front-End (Parte I)](http://www.felipefialho.com/blog/2016/do-zero-a-heroi-do-front-end-parte-1)
- [Do Zero a Herói Front-End (Parte 2)](http://www.felipefialho.com/blog/2016/do-zero-a-heroi-do-front-end-parte-2)

**Comment by vitocaio** [comment]
Opa! muito obrigado!

**Comment by vitocaio** [comment]
Teria também algum material referente a Angular JS com a mesma temática deste conteúdo que me forneceu?

**Comment by thailoeduardo** [comment]
https://github.com/ericdouglas/traduz-ai

**Comment by woliveiras** [comment]
Mais alguns artigos que podem ajudar cara:

https://nandovieira.com.br/escrevendo-javascript-modular
https://udgwebdev.com/escrevendo-javascript-melhor-parte-1
https://udgwebdev.com/escrevendo-javascript-melhor-parte-2
http://blog.da2k.com.br/2015/02/13/escalando-javascript-no-browser/

Aqui algumas dicas de carreira:

http://bit.ly/carreira-front-end

**Comment by vitocaio** [comment]
Muito obrigado a todos!
